### PR TITLE
Add function for getting events

### PR DESCRIPTION
**Pull request by jepebe**
This PR adds a function to get events from the VTN.

### CODE REVIEW

File: toadr3/events.py [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+        raise ValueError("skip must be an integer")
+
+    if skip is not None and skip < 0:
+        raise ValueError("skip must be a positive integer")
+
+    if limit is not None and not isinstance(limit, int):
+        raise ValueError("limit must be an integer")
+
+    if limit is not None and not 0 <= limit <= 50:
+        raise ValueError("limit must be between 0 and 50")
```
Comment (epa095) [inline toadr3/events.py:110]: ! Is this really a part of the spec‽‽

File: toadr3/events.py [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+        params["skip"] = skip
+
+    if limit is not None:
+        params["limit"] = limit
+
+    headers = {}
+    if access_token is not None:
+        headers["Authorization"] = f"Bearer {access_token.token}"
+
+    async with session.get(f"{vtn_url}/events", params=params, headers=headers) as response:
```
Comment (epa095) [inline toadr3/events.py:131]: Strip any "/" from the end of vtn_url, just to make it a bit friendlier?

**Review by epa095 — APPROVED** [review]
LGTM!!

File: toadr3/events.py (hunk 1 above)
Comment (jepebe) [reply toadr3/events.py:110]: Yes, but probably no...

**Pull request merged**